### DISCUSSION TITLE
Move from Snap to AIO tutorial

### DISCUSSION THREAD

**Discussion by vawaver** [nextcloud/all-in-one#2181] [Questions] [answered]
Greetings,
I would like to know whether is some tutorial how to move from existing Nextcloud Snap installation to Nextcloud AIO?
I am not professional IT so I appreciate the detailed tutorial.

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/blob/main/migration.md
